api/app/GraphQL/Validators/UpdateUserInputValidator.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\GraphQL\Validators;
+
+use Illuminate\Validation\Rule;
+use Nuwave\Lighthouse\Validation\Validator;
+
+final class UpdateUserInputValidator extends Validator
+{
+    /**
+     * Return the validation rules.
+     *
+     * @return array<string, array<mixed>>
+     */
+    public function rules(): array
+    {
+        return [
+            'email' => [
+                'sometimes',
+                Rule::unique('users', 'email')->ignore($this->arg('id'), 'id'),
```

Comment:
Could you add a comment to explain the ignore?  By next week I will have forgotten and it won't be clear to me.  :laughing:

Reply:
Yup! Hopefully what i wrote makes sense 😃 

[document ignore for unique email rule](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4103/commits/f2cbb2c152bc31ce65a0589c3cdb4865c0fc86aa)